### DISCUSSION TITLE
Can i join your tim sir?

### DISCUSSION THREAD

**Discussion by Wrdn28** [ROM-builders/temporary#18079] [Q&A]
I successfully build ricedroid 10.2 
https://cirrus-ci.com/build/5160708183162880

**This is my tree**

Device tree: https://github.com/Wrdn28/rice_device_X01AD

Vendor tree: https://github.com/Wrdn28/rice_vendor_X01AD

Kernel tree:
https://github.com/Wrdn28/kernel_asus_X01AD

My github profile
https://github.com/Wrdn28

My telegram
https://t.me/Wrdn2802

**Comment by Apon77** [comment]
please follow #team and #learn instruction in telegram
